From 0ecffb24ad20fcd4e0eea64275195e81fa10c6a8 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <b03824@freescale.com>
Date: Mon, 21 Nov 2011 09:01:39 +0800
Subject: [PATCH 212/635] ENGR00000011 fix build error on ICS

Remove the overlay header file
Use android_audio_legacy naming space for alsa module

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 alsa/alsa_imx.cpp             | 4 ++--
 mx6/hwcomposer/hwcomposer.cpp | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/alsa/alsa_imx.cpp b/alsa/alsa_imx.cpp
index 354eaed..7fd22e8 100755
--- a/alsa/alsa_imx.cpp
+++ b/alsa/alsa_imx.cpp
@@ -18,7 +18,7 @@
 #define LOG_TAG "iMXALSA"
 #include <utils/Log.h>
 
-#include "AudioHardwareALSA.h"
+#include <AudioHardwareALSA.h>
 #include <media/AudioRecord.h>
 
 #include <cutils/properties.h>
@@ -35,7 +35,7 @@
 
 #define ARRAY_SIZE(x) (sizeof(x) / sizeof(x[0]))
 
-namespace android
+namespace android_audio_legacy
 {
 
 static int s_device_open(const hw_module_t*, const char*, hw_device_t**);
diff --git a/mx6/hwcomposer/hwcomposer.cpp b/mx6/hwcomposer/hwcomposer.cpp
index 8f8bee1..051c3b8 100755
--- a/mx6/hwcomposer/hwcomposer.cpp
+++ b/mx6/hwcomposer/hwcomposer.cpp
@@ -15,7 +15,6 @@
 /*Copyright 2009-2011 Freescale Semiconductor, Inc. All Rights Reserved.*/
 
 #include <hardware/hardware.h>
-#include <hardware/overlay.h>
 
 #include <fcntl.h>
 #include <errno.h>
-- 
1.8.0

